recipe/build.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/bash
 
-# Change the permissions of the protobuf.egg-info directory to be 
-# group and world readable
-chmod 644 protobuf.egg-info/*
-$PYTHON setup.py install --single-version-externally-managed --record record.txt
+./autogen.sh
+autoconf
+autoreconf -i
+./configure --prefix=$PREFIX
+make
+
+# Cannot run `make check` on CircleCI.
+# See the PR that added this for details.
+# https://github.com/conda-forge/protobuf-feedstock/pull/5
+if [[ ! ( $CIRCLECI ) ]];
+then
+    make check
+fi
```

Comment:
We are unable to run this on CircleCI. See [log]( https://circleci.com/gh/conda-forge/protobuf-feedstock/18 ) for `make check` failure. This problem is unique to Circle CI and protobuf, but [others have suffered]( https://github.com/cockroachdb/cockroach/pull/162#issuecomment-62233187 ) from it too. The only [fix]( https://github.com/cockroachdb/cockroach/pull/162/files#diff-00c1868329bff6ba766f2caefa0451dfR50 ) appears to be not to run `make check`. So, that is what we do here.